src/components/Carousel/_carousel.scss

```diff
@@ -60,24 +60,45 @@
     }
 
     &-container-inner {
+      align-items: center;
       display: flex;
       gap: 12px;
+      max-width: var(--max-width);
+      overflow: auto;
+      -webkit-overflow-scrolling: touch;
+      -ms-overflow-style: none; /* Internet Explorer 10+ */
+      scroll-snap-type: x mandatory;
+      scrollbar-width: none; /* Firefox */
+
+      &::-webkit-scrollbar {
+        display: none; /* Safari and Chrome */
+      }
     }
 
     &-dot-container {
       align-items: center;
       display: flex;
       height: 8px;
       justify-content: center;
+      min-width: 8px;
```

Comment:
Do we need this if we are explicitly setting the width?

Reply:
there's this weird thing where the dots gets squashed without it 🤷‍♂️